src/navsections/HelmChartSectionBlueprint/HelmChartContextMenu.tsx

```diff
@@ -36,6 +37,16 @@ const StyledActionsMenuIconContainer = styled.span<{isSelected: boolean}>`
   align-items: center;
 `;
 
+// temporary solution for renaming value file
```

Comment:
let's add the `// TODO: ` prefix to this comment and maybe an explanation of what the problem might be, so we have a starting point to tackle it later